misc/python/materialize/checks/materialized_views.py

```diff
@@ -65,3 +67,113 @@ def validate(self) -> Testdrive:
            """
             )
         )
+
+
+class MaterializedViewsAssertNotNull(Check):
+    def _can_run(self, e: Executor) -> bool:
+        # CREATE ROLE not compatible with older releases
+        return self.base_version >= MzVersion.parse("0.73.0-dev")
+
+    def initialize(self) -> Testdrive:
+        return Testdrive(
+            dedent(
+                """
+                > CREATE TABLE not_null_table (x INT, y INT, z INT);
```

Comment:
I find the name `not_null_table` slightly confusing. It contains `NULL` values and has not `NOT NULL` constraints.

Reply:
It is named after the class/test.